sqlmodel/main.py

```diff
@@ -151,6 +151,10 @@ def __init__(self, default: Any = Undefined, **kwargs: Any) -> None:
                 raise RuntimeError(
                     "Passing sa_type is not supported when also passing a sa_column"
                 )
+        if not hasattr(PydanticFieldInfo, "validation_alias"):
+            kwargs.pop("validation_alias")
+        if not hasattr(PydanticFieldInfo, "serialization_alias"):
+            kwargs.pop("serialization_alias")
```

Comment:
This doesn't look necessary (`PydanticFieldInfo` seems to accept kwargs even if it doesn't use them later)